skeleton/gitops-template/components/http/base/rhoai/initialize-dsp.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       serviceAccountName: ${{ values.name }}-dsp-job
       containers:
       - name: initialize-dsp
-        image: quay.io/redhat-appstudio/appstudio-utils:5bd7d6cb0b17f9f2eab043a8ad16ba3d90551bc2@sha256:8c7fcf86af40c71aeb58e4279625c8308af5144e2f6b8e28b0ec7e795260e5f7
+        image: quay.io/redhat-ai-dev/appstudio-utils:5bd7d6cb0b17f9f2eab043a8ad16ba3d90551bc2
```

Comment:
Do we need to mirror this? 

Long term, once we move away from RHTAP, we should be maintaining our own utils image if needed, rather than mirroring redhat-appstudio/appstudio-utils

Reply:
In that case we should copy the Containerfile and content over into the dev images so we can make future changes? The tag and such could remain the same in that case?

Reply:
I'd use a name other than `appstudio-utils` (with appstudio being the original name rhtap / konflux). Perhaps just `utils`? 

But we should look at copying the container file into a new repo

Reply:
Okay we can copy the containerfile to https://github.com/redhat-ai-dev/developer-images. I'll have to rename `appstudio-utils` to whatever we choose, I am fine with `utils`, and open a PR [here](https://github.com/redhat-ai-dev/ai-lab-app) first and then I can get those changes into here to reduce back and forth.